packages/account/src/Sections/Security/ConnectedApps/connected-apps.scss

```diff
@@ -126,6 +125,10 @@
         border-bottom: 1px solid var(--general-section-1);
     }
 
+    .table__head .table__row {
+        font-weight: bold;
+    }
```

Comment:
Can we handle this using `weight` prop on Text component?

Reply:
- Hey @matin-deriv this is a common component `<DataTable>` that doesn't pass the style as a prop, so I used this to overwrite the style for this particular case.